environments/typescript/recommended.js

```diff
@@ -49,93 +49,100 @@ module.exports = {
 
     // Require that member overloads be consecutive
     // Grouping overloaded members together can improve readability of the code.
-    'typescript/adjacent-overload-signatures': 'warn',
+    '@typescript-eslint/adjacent-overload-signatures': 'warn',
 
     // Require PascalCased class and interface names
     // This rule aims to make it easy to differentiate classes from regular variables at a glance.
-    'typescript/class-name-casing': 'warn',
+    '@typescript-eslint/class-name-casing': 'warn',
 
     // Require explicit return types on functions and class methods
     // Explicit types for function return values makes it clear to any calling code what type is
     // returned. This ensures that the return value is assigned to a variable of the correct type;
     // or in the case where there is no return value, that the calling code doesn't try to use the
     // undefined value when it shouldn't.
-    'typescript/explicit-function-return-type': ['warn', {
+    '@typescript-eslint/explicit-function-return-type': ['warn', {
       allowExpressions: true,
     }],
 
     // Require explicit accessibility modifiers on class properties and methods
     // This rule aims to make code more readable and explicit about who can use which properties.
-    'typescript/explicit-member-accessibility': 'warn',
+    '@typescript-eslint/explicit-member-accessibility': 'warn',
 
     // Require that interface names be prefixed with I
     // It can be hard to differentiate between classes and interfaces. Prefixing interfaces with "I"
     // can help telling them apart at a glance.
-    'typescript/interface-name-prefix': ['warn', 'always'],
+    '@typescript-eslint/interface-name-prefix': ['warn', 'always'],
 
     // Require a specific member delimiter style for interfaces and type literals
     // This rule aims to standardise the way interface and type literal members are delimited.
-    'typescript/member-delimiter-style': ['warn', {
-      delimiter: 'none',
+    '@typescript-eslint/member-delimiter-style': ['warn', {
+      multiline: {
+        delimiter: 'none',
+        requireLast: false
```

Comment:
Is this change necessary in order to switch the packages or is this a rule change suggestion? 😇 If it is the latter I would prefer to have it in a separate PR so we can discuss the change with the whole team. 🙏

Reply:
It's a necessary change. It should behave in the same way as before 😇